src/components/Shared/Filters/FilterListItemDateRange.tsx

```diff
@@ -0,0 +1,72 @@
+import { ListItem, ListItemText } from '@material-ui/core';
+import { DatePicker } from '@material-ui/pickers';
+import React from 'react';
+import { DateTime } from 'luxon';
+import { useTranslation } from 'react-i18next';
+import { Filter } from './Filter';
+
+interface Props {
+  filter: Filter;
+  value;
+  onUpdate: (value: string) => void;
+}
+
+export const FilterListItemDateRange: React.FC<Props> = ({
+  filter,
+  value,
+  onUpdate,
+}) => {
+  const { t } = useTranslation();
+
+  const [startDate, endDate] = ((value: string | null) =>
+    value?.split('..', 2)?.map((value) => DateTime.fromISO(value) || null) ?? [
+      null,
+      null,
+    ])(value);
```

Comment:
Would using intervals work for this? Looks like we might have to replace `..` with a `/`. https://moment.github.io/luxon/docs/class/src/interval.js~Interval.html#static-method-fromISO.

Also it's odd that this is a IIFE.